Ical.Net/Constants.cs

```diff
@@ -219,6 +219,7 @@ public enum FrequencyOccurrence
         Fifth = 5
     }
 
+    [Obsolete("Usage may cause undesired results or exceptions. Will be removed.", false)]
```

Comment:
We receive warnings at usage sites now (e.g. RecurrrencePatternEvaluator). We should suppress those warnings there.

Reply:
right, will add #pragma warning disable 0618 where needed